mlrun/db/httpdb.py

```diff
@@ -653,8 +653,7 @@ def read_artifact(self, key, tag=None, iter=None, project=""):
         tag = tag or "latest"
         endpoint_path = f"projects/{project}/artifacts/{key}?tag={tag}"
         error = f"read artifact {project}/{key}"
-        # The default is legacy format, need to override it.
-        params = {"format": schemas.ArtifactsFormat.full.value}
+        params = {}
```

Comment:
I would keep the `full` value here, just to make sure it works properly if the server is older (not really expected, but still) - there's no harm in being explicit here.